hack/test-cmd.sh

```diff
@@ -246,112 +246,16 @@ os::test::junit::declare_suite_start "cmd/config"
-
-# log in as an image-pruner and test that oadm prune images works against the atomic binary
-os::cmd::expect_success "oadm policy add-cluster-role-to-user system:image-pruner pruner --config='${MASTER_CONFIG_DIR}/admin.kubeconfig'"
-os::cmd::expect_success "oc login --server=${KUBERNETES_MASTER} --certificate-authority='${MASTER_CONFIG_DIR}/ca.crt' -u pruner -p anything"
-# this shouldn't fail but instead output "Dry run enabled - no modifications will be made. Add --confirm to remove images"
-os::cmd::expect_success 'oadm prune images'
-
-# log in and set project to use from now on
-VERBOSE=true os::cmd::expect_success "oc login --server=${KUBERNETES_MASTER} --certificate-authority='${MASTER_CONFIG_DIR}/ca.crt' -u test-user -p anything"
-VERBOSE=true os::cmd::expect_success 'oc get projects'
-VERBOSE=true os::cmd::expect_success 'oc project project-foo'
-os::cmd::expect_success_and_text 'oc config view' "current-context.+project-foo/${API_HOST}:${API_PORT}/test-user"
-os::cmd::expect_success_and_text 'oc whoami' 'test-user'
-os::cmd::expect_success_and_text "oc whoami --config='${MASTER_CONFIG_DIR}/admin.kubeconfig'" 'system:admin'
-os::cmd::expect_success_and_text 'oc whoami -t' '.'
-os::cmd::expect_success_and_text 'oc whoami -c' '.'
-
-# test config files from the --config flag
-os::cmd::expect_success "oc get services --config='${MASTER_CONFIG_DIR}/admin.kubeconfig'"
-
-# test config files from env vars
-os::cmd::expect_success "KUBECONFIG='${MASTER_CONFIG_DIR}/admin.kubeconfig' oc get services"
-
-# test completion command help
-os::cmd::expect_success_and_text "oc completion -h" "prints shell code"
-os::cmd::expect_success_and_text "openshift completion -h" "prints shell code"
-os::cmd::expect_success_and_text "oadm completion -h" "prints shell code"
-# test completion command output
-os::cmd::expect_failure_and_text "oc completion" "Shell not specified."
-os::cmd::expect_success "oc completion bash"
-os::cmd::expect_success "oc completion zsh"
-os::cmd::expect_failure_and_text "oc completion test_shell" 'Unsupported shell type "test_shell"'
-# test completion command for openshift
-os::cmd::expect_failure_and_text "openshift completion" "Shell not specified."
-os::cmd::expect_success "openshift completion bash"
-os::cmd::expect_success "openshift completion zsh"
-os::cmd::expect_failure_and_text "openshift completion test_shell" 'Unsupported shell type "test_shell"'
-# test completion command for oadm
-os::cmd::expect_failure_and_text "oadm completion" "Shell not specified."
-os::cmd::expect_success "oadm completion bash"
-os::cmd::expect_success "oadm completion zsh"
-os::cmd::expect_failure_and_text "oadm completion test_shell" 'Unsupported shell type "test_shell"'
-echo "oc completion: ok"
-
-# test config files in the home directory
-mkdir -p ${HOME}/.kube
-cp ${MASTER_CONFIG_DIR}/admin.kubeconfig ${HOME}/.kube/config
-os::cmd::expect_success 'oc get services'
-mv ${HOME}/.kube/config ${HOME}/.kube/non-default-config
```

Comment:
@deads2k claims that the double-move with `oc get services` in between would have done something... but there was no diff, no comments and no affirmative test for changes so I got rid of this